lib/airbrake-ruby/backtrace.rb

```diff
@@ -188,6 +193,21 @@ def match_frame(regexp, stackframe)
 
         Patterns::GENERIC.match(stackframe)
       end
+
+      def read_code_hunk(logger, frame)
+        code_hunk = Airbrake::CodeHunk.new(frame[:file], frame[:line]).to_h
+        return unless code_hunk
+
+        unless code_hunk.key?(0)
+          frame[:code_hunk] = code_hunk
+          return
+        end
+
+        logger.error(
+          "#{LOG_LABEL} error while reading code hunk `#{file}:#{line}'. " \
+          "Reason: #{code_hunk[0]}"
```

Comment:
Again, I don't understand why `code_hunk[0]` contains the reason.

Reply:
Files can never have line 0.